topic/topic.go

```diff
@@ -137,6 +138,21 @@ func (t *topic) RemoveConsumerService(value services.ServiceID) (Topic, error) {
 	return nil, fmt.Errorf("could not find consumer service %s in the topic", value.String())
 }
 
+func (t *topic) UpdateConsumerService(value ConsumerService) (Topic, error) {
```

Comment:
This is going to be called from a cli command?

Reply:
yep